vtex-apps/add-to-cart-button
# Feat/bpf1427

#### What problem is this solving? 
updated affiliate key to affiliation



#### How to test it?
on the PDP.. when you click on the button add to cart.. open up the console. and type window.dataLayer and in the list of events you will see add to cart.. the key should be affiliation and it should have a value 


[Workspace](Link goes here!)

#### Screenshots or example usage:
<img width="674" alt="Screenshot 2023-07-05 at 11 15 48" src="https://github.com/vtex-apps/add-to-cart-button/assets/27976086/d5b41162-5d69-4601-a9a7-e080255834b5">



#### Describe alternatives you've considered, if any.



#### Related to / Depends on



#### How does this PR make you feel? [:link:](http://giphy.com/)



![](put .gif link here - can be found under "advanced" on giphy)
